capture/engine_scripts/cookies.json

```diff
@@ -1,6 +1,5 @@
 [
   {
-    "domain": ".www.yourdomain.com",
```

Comment:
Only problem I have here is that this property will be there when using popular plugins to create this JSON file.   So, I prefer to leave this as I imagine many users will include this.